graylog2-server/src/test/java/org/graylog2/configuration/IndexerDiscoveryProviderTest.java

```diff
@@ -56,6 +56,23 @@ void testAutomaticDiscovery() {
     }
 
 
+    @Test
+    void testAutomaticDiscoveryOneUnconfigured() {
```

Comment:
I'd just rename the test method to a more precise one, like `testDiscoveryIgnoresEmptyAddresses`